src/shared/constants/index.ts

```diff
@@ -7,4 +7,8 @@ export {
 	WYSIWYG_OPTIONS_ALIGN,
 	WYSIWYG_OPTIONS_FULL_WITHOUT_ALIGN,
 } from './wysiwyg';
+
 export const DEFAULT_AUDIO_STILL = '/images/audio-still.svg';
+
+export const NOT_NOW_KEY = 'ACMIDMNudgeModal.NotNow';
```

Comment:
If this key name can be anything we want, consider following the existing naming convention:
```
export const STAMBOEK_LOCAL_STORAGE_KEY = 'AVO.stamboek';
export const TOUR_DISPLAY_DATES_LOCAL_STORAGE_KEY = 'AVO.tour_display_dates';
```